_tools/README.md

```
@@ -0,0 +1,74 @@
+
+#### 1. Git Bash 설치
+
+[https://git-scm.com/download/win](https://git-scm.com/download/win)  
+설치 중 "Git Bash Here" 옵션을 포함하도록 진행합니다.
+
+#### 2. Android Studio 설정
+
+- 메뉴: `File > Settings > Tools > Terminal`
+- **Shell path**를 아래로 설정:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)